yt/yt/tests/integration/queues/test_queue_agent.py

```diff
@@ -2599,11 +2599,12 @@ def try_remove():
         wait(try_remove, ignore_exceptions=True)
 
     @staticmethod
-    def _check_export(export_directory, expected_data, queue_path=None):
+    # NB: The last two options should be used carefully: currently they strictly check that all timestamps are within [ts - period, ts], which might not generally be the case.
+    def _check_export(export_directory, expected_data, queue_path=None, use_upper_bound_for_table_names=True, export_period=None):
```

Comment:
Let's change default of `use_upper_bound_for_table_names` to `false` as it is in server code.